ansible/inventory.yml

```
@@ -267,8 +270,11 @@ hosts:
             user: administrator
 
     - iinthecloud:
-        ibmi72-ppc64_be-1: {ip: 65.183.160.52, user: nodejs}
-        ibmi72-ppc64_be-2: {ip: 65.183.160.59, user: nodejs}
+        ibmi73-ppc64_be-1: {ip: 65.183.160.52, user: nodejs}
+        ibmi73-ppc64_be-2: {ip: 65.183.160.59, user: nodejs}
+
+    - rzkh:
+        ibmi73-ppc64_be-4: {ip: 185.113.4.148, user: nodejs}
```

```suggestion
        ibmi73-ppc64_be-1: {ip: 185.113.4.148, user: nodejs}
```
The hostnames we use in Jenkins follow `<type>-<provider>-<os>-<arch>-<num>` and we usually start numbering from 1, e.g. `test-rzkh-ibmi73-ppc64_be-1`.